.github/workflows/workflow.yml

```diff
@@ -45,13 +48,52 @@ jobs:
         python-version: ${{ matrix.python-version }}
     - name: Install dependencies
       run: |
-        python -m pip install --upgrade pip
         pip install tox tox-gh-actions
     - name: Test with tox
       run: tox --skip-missing-interpreters false -e py${{ matrix.python-version }}-${{ matrix.pyenv }}
     - name: Upload Coverage to CodeCov
       uses: codecov/codecov-action@v3
 
+  conda-build:
+    needs: check
+    runs-on: ${{ matrix.os }}
+    # Only run on review ready pull_requests
+    if: ${{ github.event_name == 'pull_request' && github.event.pull_request.draft == false }}
+    defaults:
+      run:
+        shell: bash -el {0}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, ubuntu-20.04, macOS-latest, macOS-11, windows-latest]
+        backend: [default, openblas]
+        python-version: ['3.8']
+        include:
+          - os: ubuntu-latest
+            backend: intel
+          - os: macOS-latest
+            backend: accelerate
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Conda ${{ matrix.os }} Python ${{ matrix.python-version }}
+      uses: conda-incubator/setup-miniconda@v2
+      with:
+        auto-update-conda: true
+        python-version: ${{ matrix.python-version }}
+        activate-environment: aspire
+        environment-file: environment-${{ matrix.backend }}.yml
+        auto-activate-base: false
+    - name: Complete Install and Log Environment ${{ matrix.os }} Python ${{ matrix.python-version }}
+      run: |
+        conda info
+        conda list
+        pip install -e ".[dev]"
+        pip freeze
+        python -c "import numpy; numpy.show_config()"
+    - name: Execute Pytest Conda ${{ matrix.os }} Python ${{ matrix.python-version }}
+      run: |
+        export OMP_NUM_THREADS=2
+        pytest -n2 --durations=50 -s
```

Comment:
`-n2`? Couldn't find any docs on this.

Reply:
That's for `pytest-xdist`. It runs the pytests in parallel. Shaves a little time.  (Try it locally somtime say -n3 or 4).

Added a comment.

Reply:
Got it. Thanks.